save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/projects/ProjectDemoMenu.kt

```diff
@@ -222,7 +229,7 @@ private fun ChildrenBuilder.renderFileUploading(
                 autoComplete = AutoComplete.off
                 placeholder = "Release tag"
                 value = demoDto.vcsTagName
-                disabled = demoStatus != DemoStatus.NOT_CREATED && demoStatus != DemoStatus.STOPPED
+                disabled = true
```

Comment:
is it a temporary disabled?

Reply:
Yes, it is disabled until we allow to choose between GitHub download and manual download